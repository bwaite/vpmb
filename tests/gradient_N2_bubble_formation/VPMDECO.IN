TRIMIX DIVE TO 40 MSW
1
.15,.45,.40
1
0,40,23,1
2
40,30,1
99
1
40,1,-10,3
0
