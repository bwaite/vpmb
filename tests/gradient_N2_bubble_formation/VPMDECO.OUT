                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    03-03-2010 at 08:26 pm                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 40 MSW                                                 
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (mswg)  (mswg)  (msw/min) |  (mswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     1.7    1.7 |    1   | Descent     0.     40.      23.0   |
  2    28.3   30.0 |    1   |                                    |    40.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at   33.3 mswg
                 Deepest possible decompression stop is   33.0 mswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (mswg) (msw/min)  Used  | (mswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  3     1.3   31.3 |    1   |   27.    -10.0          |
  4     0.7   32.0 |    1   |                         |    27      1     32
  5     0.3   32.3 |    1   |   24.    -10.0          |
  6     0.7   33.0 |    1   |                         |    24      1     33
  7     0.3   33.3 |    1   |   21.    -10.0          |
  8     2.7   36.0 |    1   |                         |    21      3     36
  9     0.3   36.3 |    1   |   18.    -10.0          |
 10     3.7   40.0 |    1   |                         |    18      4     40
 11     0.3   40.3 |    1   |   15.    -10.0          |
 12     5.7   46.0 |    1   |                         |    15      6     46
 13     0.3   46.3 |    1   |   12.    -10.0          |
 14     8.7   55.0 |    1   |                         |    12      9     55
 15     0.3   55.3 |    1   |    9.    -10.0          |
 16    16.7   72.0 |    1   |                         |     9     17     72
 17     0.3   72.3 |    1   |    6.    -10.0          |
 18    34.7  107.0 |    1   |                         |     6     35    107
 19     0.3  107.3 |    1   |    3.    -10.0          |
 20   141.7  249.0 |    1   |                         |     3    142    249
 21     0.3  249.3 |    1   |    0.    -10.0          |
 
